src/App.js

```diff
@@ -1,23 +1,14 @@
-import logo from './logo.svg';
+import React from 'react';
+import {BrowserRouter as Router, Routes, Route, Link} from 'react-router-dom';
 import './App.css';
+import HomePage from './Pages/HomePage';
```

Comment:
Since the file HomePage is under the folder “Pages”, naming it “Home**Page**” seems redundant. Consider naming “Home.js” instead. I’d recommend following the same pattern for all other files inside the “Pages” folder.